src/SFML/Network/Packet.cpp

```diff
@@ -29,12 +29,57 @@
 #include <SFML/Network/SocketImpl.hpp>
 #include <SFML/System/String.hpp>
 
+#include <algorithm>
+#include <array>
 #include <cstring>
 #include <cwchar>
 
 
 namespace sf
 {
+namespace
+{
+////////////////////////////////////////////////////////////
+/// \brief Endian test
+///
+////////////////////////////////////////////////////////////
+inline bool isBigEndian()
+{
+    int one = 1;
+    return (*reinterpret_cast<char*>(&one)) != 1;
+};
+
+
+////////////////////////////////////////////////////////////
+/// \brief Generic endianness conversion; if host is big-endian,
+/// incoming network byte order can be handled without swap, and
+/// host byte order is already network byte order
+///
+////////////////////////////////////////////////////////////
+template <typename T>
+T swapEndian(T val)
```

Comment:
```suggestion
T swapEndian(T value)
```
There's no need to abbreviate words that are already 5 characters. We allow for 120 character columns so vertical space is not a precious resource. Same goes for other abbreviated variable names. Please be descriptive.